resources/local/chainspec.toml.in

```diff
@@ -73,6 +73,29 @@ simultaneous_peer_requests = 5
 consensus_protocol = 'Highway'
 # The maximum amount of delegators per validator. if the value is 0, there is no maximum capacity.
 max_delegators_per_validator = 0
+# Allow auction entrypoints such as delegate, redelegate and add_bid to function properly.
+#
+# Changing this option makes sense only for private chains which dont need auctioning new validator slots.
+allow_auction_bids = true
+# Allow peer to peer transfers between users. Setting this to false makes sense only on private chains.
+allow_unrestricted_transfers = true
```

Comment:
This would be clearer as `restrict_transfers = false` IMO, but I know this is a merge PR and I guess we shouldn't make a change like this here.  Can we ticket this for future work though?

Reply:
#4164